<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FractalParameters</class>
 <widget class="QWidget" name="FractalParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>370</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Formula:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="formula">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Interior color:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="interiorColor"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Exterior color:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="exteriorColor"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Plane:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="plane"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Perturbation (Re):</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Perturbation (Im):</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Center (Re):</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="centerX"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Center (Im):</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="centerY"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="radius"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Angle</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Iterations</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Bailout</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButton_2">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="toolButton_3">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="toolButton_4">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
   </item>
   <item row="10" column="1">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="9" column="1">
    <widget class="QSpinBox" name="spinBox_2"/>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="perturbationX">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSlider" name="perturbationY">
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>formula</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>interiorColor</tabstop>
  <tabstop>exteriorColor</tabstop>
  <tabstop>plane</tabstop>
  <tabstop>radius</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
